charts/karpenter/values.yaml

```diff
@@ -145,6 +145,17 @@ postInstallHook:
     tag: "1.30"
     # -- SHA256 digest of the post-install hook image.
     digest: sha256:13a2ad1bd37ce42ee2a6f1ab0d30595f42eb7fe4a90d6ec848550524104a1ed6
+  # -- Resources for the post-install-hook pod.
+  resources: {}
```

Comment:
I can't imagine that this is going to be variable. Can we just set this to static resource values and not need configurable values?

Reply:
This is a best practice. Resources are never hardcoded.

E.g. Depending on a specific cluster the requirements for resources might differ. One cluster might just have 1 nodepool where another one might have many more where more resources might be allocated to perform the task.